composer.json

```
@@ -0,0 +1,45 @@
+{
+    "name": "keeleinstituut/tv-common-laravel-security",
+    "description": "Authorization lib of Tõlkevärav",
+    "keywords": [
+        "laravel",
+        "keycloak"
+    ],
+    "require": {
+        "firebase/php-jwt": "^6.3",
+        "guzzlehttp/guzzle": "^7.5",
+        "php": "^8.2"
+    },
+    "autoload": {
+        "psr-4": {
+            "KeycloakAuthGuard\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "KeycloakAuthGuard\\Tests\\": "tests/"
+        }
+    },
+    "scripts": {
+        "test": "pest"
+    },
+    "extra": {
+        "laravel": {
+            "providers": [
+                "KeycloakAuthGuard\\KeycloakAuthServiceProvider"
+            ]
+        }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^10.0",
+        "orchestra/testbench": "^8.3",
+        "friendsofphp/php-cs-fixer": "^3.10",
```

Hmm, since our other PHP services follow the [Pint codestyle](https://github.com/laravel/pint) (included by default in Laravel), perhaps we ought to use that here as well (instead of CS Fixer)? Pint is built on top of CS Fixer.

```suggestion
        "laravel/pint": "^1.9",
```